grammars/javascript.cson

```diff
@@ -10,6 +10,7 @@
   'pjs'
   'xsjs'
   'xsjslib'
+  'jscad'
```

Comment:
It doesn't look like that list is in alphabetical order. If it was in alphabetical order it would look like:
``` coffeescript
'fileTypes': [
  '_js'
  'es'
  'es6'
  'htc'
  'js'
  'jscad'
  'jsm'
  'pjs'
  'xsjs'
  'xsjslib'
]
```

Should I just realphabetize the entire list?